test/unit/refresh.js

```diff
@@ -22,7 +22,7 @@ var fs = require('fs');
 var format = require('util').format;
```

Comment:
Sorry, I re-formatted this file, so it has a huge diff. I added just a single test at line 1349.